docs/product-installation-guide/src/main/asciidoc/run-standalone-properties-con.adoc

```diff
@@ -1,5 +1,12 @@
-[id='install-standalone-properties-con']
+[id='run-standalone-properties-con']
 = Supported properties
+When you install standalone {CENTRAL}, you can use the properties listed in this section in the following command:
+[source] 
+----
+java -jar rhdm-7.0.0.GA-decision-central-standalone.jar -s application-config.yaml -D<property>=<value> -D<property>=<value>
+----
+In this command, `<property>` is a property from the following list and `<value>` is a value that you assign to that property.
+
 * `org.uberfire.nio.git.dir`: Location of the directory .niogit. Default: working directory
```

Comment:
Some recurring style issues (per RH/IBM style) in this list of property descriptions:
- Put file/folder/config names in unicode quotes (`.niogit`, `.index`, `org.uberfire.admin`, etc.)
- Cap SSH throughout (some are and some aren't capped)
- Change the ones that start "Disable ..." to "Disables ..." for consistency.